Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/CompositorWindow.cs

```diff
@@ -0,0 +1,346 @@
+                        }
+                    }
+                    else
+                    {
+                        if (GUILayout.Button("Stop Recording"))
+                        {
+                            compositionManager.StopRecording();
+                        }
+                    }
+                }
+
+                if (GUILayout.Button("Take Picture"))
+                {
+                    compositionManager.TakePicture();
+                }
+
+                EditorGUILayout.Space();
+                GUI.enabled = true;
+
+                // Open Folder
+                if (GUILayout.Button("Open Folder"))
+                {
+                    Process.Start(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "HologramCapture"));
+                }
+
+                EditorGUILayout.EndVertical();
+            }
+        }
+
+        private void HologramSettingsGUI()
+        {
+            hologramSettingsFoldout = EditorGUILayout.Foldout(hologramSettingsFoldout, "Hologram Settings");
+            if (hologramSettingsFoldout)
+            {
+
+                EditorGUILayout.BeginVertical("Box");
+                {
+                    CompositionManager compositionManager = GetCompositionManager();
+
+                    if (compositionManager != null)
+                    {
+                        EditorGUILayout.Space();
+
+                        GUI.enabled = compositionManager == null || !compositionManager.IsVideoFrameProviderInitialized;
+                        compositionManager.CaptureDevice = (CompositionManager.FrameProviderDeviceType)EditorGUILayout.Popup("Video source", ((int)compositionManager.CaptureDevice), Enum.GetNames(typeof(CompositionManager.FrameProviderDeviceType)));
+                        GUI.enabled = true;
+
+                        EditorGUILayout.Space();
+                    }
+
+                    float newAlpha = EditorGUILayout.Slider("Alpha", hologramAlpha, 0, 1);
+                    if (newAlpha != hologramAlpha)
+                    {
+                        hologramAlpha = newAlpha;
+                        if (compositionManager != null && compositionManager.TextureManager != null)
+                        {
+                            compositionManager.TextureManager.SetHologramShaderAlpha(newAlpha);
+                        }
+                    }
+
+                    EditorGUILayout.Space();
+
+                    if (compositionManager != null)
+                    {
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            float previousFrameOffset = compositionManager.VideoTimestampToHolographicTimestampOffset;
+                            compositionManager.VideoTimestampToHolographicTimestampOffset = EditorGUILayout.Slider("Frame time adjustment", previousFrameOffset, -1 * maxFrameOffset, maxFrameOffset);
+                        }
```

Comment:
We may want to try and use GUIContent for some of these items so that we can add tooltips:
[https://docs.unity3d.com/ScriptReference/GUIContent.html](https://docs.unity3d.com/ScriptReference/GUIContent.html)

I'll admit that I haven't tested this myself. :)